pkg/apis/core/v1alpha1/federatedservicestatus_types.go

```diff
@@ -0,0 +1,41 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+import (
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// FederatedServiceClusterStatus is the observed status of the resource for a named cluster
+type FederatedServiceClusterStatus struct {
+	ClusterName string               `json:"clusterName,omitempty"`
+	Status      corev1.ServiceStatus `json:"status,omitempty"`
+}
+
+// +genclient
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+// FederatedServiceStatus
+// +k8s:openapi-gen=true
+// +kubebuilder:resource:path=federatedservicestatuses
+type FederatedServiceStatus struct {
+	metav1.TypeMeta   `json:",inline"`
+	metav1.ObjectMeta `json:"metadata,omitempty"`
+
+	ClusterStatus []FederatedServiceClusterStatus `json:"clusterStatus,omitempty"`
```

Comment:
I had resigned myself to `StatusStatus`, so I'm glad to see this alternative.  I think it makes sense to differentiate the representation of a non-configuration resource from the standard spec/status sections.  I'm curious to hear what others think.